mpc_follower_wrapper/CMakeLists.txt

```diff
@@ -0,0 +1,216 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(mpc_follower_wrapper)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+add_compile_options(-std=c++11)
+set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
+add_definitions(-DBOOST_LOG_DYN_LINK=1)
+set(CMAKE_BUILD_TYPE Debug)
+set(CMAKE_CXX_FLAGS "-lboost_system") 
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  cav_msgs
+  roscpp
+  rospy
+  std_msgs
+  geometry_msgs
+  autoware_msgs
+  autoware_config_msgs
```

Comment:
This message set is never used in this package and can be removed from here the code and package.xml